Modules/getpath.py

```diff
@@ -692,6 +692,18 @@ def search_up(prefix, *landmarks, test=isfile):
         if stdlib_dir:
             pythonpath.append(stdlib_dir)
         if platstdlib_dir:
+            if os_name == 'nt' and real_executable_dir and \
+                    isfile(joinpath(real_executable_dir, BUILDDIR_TXT)):
+                p = platstdlib_dir
```

Comment:
This line doesn't seem to do anything since you set `p` in both branches of the try .. except block.